src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -31,9 +31,9 @@ Before using [the user migration wizard](#find), please review the guidelines be
 Requirements and guidelines: 
 
 * You must have the [Owner role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#roles) to start using the user migration wizard. 
-* If your organization has more than 20 accounts in it, we recommend **not** using the wizard at this point in time.  
-* If your organization has a fairly complex parent/sub-account structure, we recommend **not** using the wizard at this point in time. 
-* If you frequently add accounts to your organization, we recommend **not** using the wizard at this point in time. On the new model, there isn't yet a way to add accounts on your own. [Learn more about adding accounts.](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#creating)
+* If your organization has more than 20 accounts, we recommend **not** using the wizard at this point in time. Note that this refers to having 20 or more accounts, not users.
+* If your organization has a fairly complex master/sub-account structure, we recommend **not** using the wizard at this point in time. 
```

Comment:
I assume we're keeping "master/sub-account" because this is referring to an historical usage?

Reply:
actually we can't yet change it, might be a few more weeks